plugins/synced_folders/rsync/helper.rb

```diff
@@ -67,6 +67,11 @@ def self.rsync_single(machine, ssh_info, opts)
         # Folder options
         opts[:owner] ||= ssh_info[:username]
         opts[:group] ||= ssh_info[:username]
+        
+        # On freebsd the root user belongs to the wheel group
+        if machine.guest.name == :freebsd && ssh_info[:username] == 'root'
+          opts[:group] = 'wheel'
```

Comment:
We should log to the user that we changed the group here so that it's clear what is going on in debug mode:

You'll need to make sure our logging library is loaded at the top....

```ruby
require 'log4r'
```

And we need to initialize a logger object:

```ruby
def self.rsync_single(.... )
  logger = Log4r::Logger.new("vagrant::synced_folders::rsync::helper::rsync_single")
  ...
  ...
```

Then now back here in this if-statement after updating `opts[:group]` we can log to debug that we're updating the group:

```ruby
logger.debug("FreeBSD guest detected, updating guests destination shared folder group to 'wheel' instead of 'root'")
```